src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -18,6 +18,7 @@ public enum IamAction {
   UPDATE_AUTH_DOMAIN,
   ADMIN_READ_SUMMARY_INFORMATION,
   // datarepo (admin-only actions)
+  ADMIN_TOGGLE_INHERIT_STEWARD,
```

Comment:
```suggestion
  TOGGLE_INHERIT_STEWARD,
```
I think this has  to match the Iam [action name in sam](https://github.com/broadinstitute/sam/blob/858e7e27ca2de75b9c3403e7f2d5907aef3cc732/src/main/resources/reference.conf#L1347)